rplugin/python3/denite/source/command.py

```diff
@@ -0,0 +1,54 @@
+# ============================================================================
+# FILE: command.py
+# AUTHOR: Shougo Matsushita <Shougo.Matsu at gmail.com>
+#         Luis Carlos Cruz Carballo <lcruzc at linkux-it.com>
+# License: MIT license
+# ============================================================================
+
+import re
+
+from .base import Base
+
+
+re_command = re.compile(r'^\|:.+\|')
```

Comment:
I think is better as global it will never change and also won't be created per instance the regular expression. Otherwise is not a problem memory perfomance.